clang/docs/HIPSupport.rst

```diff
@@ -175,6 +175,8 @@ Predefined Macros
      - Defined when the GPU default stream is set to per-thread mode.
    * - ``HIP_API_PER_THREAD_DEFAULT_STREAM``
      - Alias to ``__HIP_API_PER_THREAD_DEFAULT_STREAM__``. Deprecated.
+   * - ``__AMDGCN_WAVEFRONT_SIZE__``
```

Comment:
this macro and other AMDGPU predefined macros are documented at https://clang.llvm.org/docs/AMDGPUSupport.html

we should not just mention the link and talk about their dependency on specific GPU arch and taking default value in host compilation.

Reply:
do these really need to be defined for the host?

Reply:
No, it's explicitly wrong, but there's a lot of HIP headers that depend on it for some reason so we can't break it.